src/Components/CloudAccounts/CloudAccountsTable.tsx

```diff
@@ -0,0 +1,126 @@
+  cloudAccountsProviderFilterData,
+} from '../../state/cloudAccounts';
+
+type Props = {
+  cloudAccounts: CloudAccount[];
+};
+
+export const CloudAccountsTable = ({ cloudAccounts }: Props) => {
+  const cloudAccountFilter = useAtomValue(cloudAccountsAccountFilterData);
+  const cloudProviderFilter = useAtomValue(cloudAccountsProviderFilterData);
+  const cloudGoldImageFilter = useAtomValue(cloudAccountsGoldImageFilterData);
+
+  const rows: CloudAccountRow[] = cloudAccounts
+    .filter((acct) => {
+      if (
+        cloudAccountFilter.length > 0 &&
+        !cloudAccountFilter.some((val) =>
+          acct.providerAccountID.toLowerCase().includes(val.toLowerCase())
+        )
+      ) {
+        return false;
+      }
+
+      if (
+        cloudProviderFilter.length > 0 &&
+        !cloudProviderFilter.includes(PROVIDER_MAP[acct.shortName])
+      ) {
+        return false;
+      }
+
+      if (
+        cloudGoldImageFilter.length > 0 &&
+        !cloudGoldImageFilter.includes(acct.goldImageAccess)
+      ) {
+        return false;
+      }
+
+      return true;
+    })
+    .map((acct) => ({
+      id: acct.providerAccountID,
+      provider: PROVIDER_MAP[acct.shortName],
+      goldImage: acct.goldImageAccess,
+      date: acct.dateAdded,
+    }));
+
+  const { sorted, getSortParams } = useTableSort(rows, 'cloudAccounts', {
+    rowTranslator: (row) => [row.id, row.provider, row.goldImage, row.date],
+    initialSort: {
+      index: 0,
+      dir: SortByDirection.asc,
+    },
+  });
+  return (
+    <Table aria-label="Cloud accounts table" variant="compact">
+      <Thead>
+        <Tr>
+          <Th sort={getSortParams(0)}>Cloud account</Th>
+          <Th sort={getSortParams(1)}>Cloud provider</Th>
+          <Th sort={getSortParams(2)}>Gold image access</Th>
+          <Th sort={getSortParams(3)}>Date added</Th>
+          <Th screenReaderText="Actions" />
+        </Tr>
+      </Thead>
+      <Tbody>
+        {sorted.map((row) => (
+          <Tr key={row.id}>
+            <Td>
+              <Button variant="link" isInline component="a" href="">
+                {row.id}
+              </Button>
+            </Td>
+
+            <Td>
+              <Button
+                variant="link"
+                isInline
+                component="a"
+                href={`gold-images?provider=${providerToParam[row.provider]}`}
+              >
+                {row.provider}
+              </Button>
+            </Td>
+
+            <Td>
+              <span className="pf-v6-u-display-flex pf-v6-u-align-items-center">
+                {getStatusIcon(row.goldImage, `Status: ${row.goldImage}`)}
+                <Content className="pf-v6-u-ml-sm">{row.goldImage}</Content>
+              </span>
+            </Td>
+
+            <Td>{formatDate(row.date)}</Td>
+
+            <Td>
+              <Button variant="link" isInline component="a">
+                View Purchases
+              </Button>
+            </Td>
```

Comment:
AC: This link was to be added in a later story, [TUSC-380](https://issues.redhat.com/browse/TUSC-380). Should be fine to leave it in now as an empty link though and just update it later.